js/generic.js

```diff
@@ -514,11 +444,11 @@ function get_isVer(METRIC) {
 	return
 
 	function cascade() {
+		try {if ("lij" == Intl.PluralRules.supportedLocalesOf("lij").join()) {return 134}} catch(e) {} // 1927706
```

Comment:
This check triggers for me on FF 132 (alpine linux). How is this supposed to work?

Reply:
#134